### PR TITLE
power: Add HomeSeer control

### PR DESCRIPTION
**Pull request by Doridian**
Adds HomeSeer control

Example config:
```
[power printer]
type: homeseer
address: homeseer.local
device: 42
user: admin
password: abcdefgh
```

### CODE REVIEW

File: moonraker/plugins/power.py [hunk 1]
```diff
@@ -522,6 +524,62 @@ async def set_power(self, state):
...
+    def __init__(self, config):
+        super().__init__(config)
+        self.server = config.get_server()
+        self.addr = config.get("address")
+        self.device = config.getint("device")
+        self.user = config.get("user", "admin")
+        self.password = config.get("password", "")
+
+    async def _send_homeseer(self, request, additional = ""):
+        url = f"http://{self.user}:{self.password}@{self.addr}/JSON?user={self.user}&pass={self.password}&request={request}&ref={self.device}&{additional}"
```
Comment (Arksine) [inline moonraker/plugins/power.py:537]: This line should be wrapped to no more than 80 chars.

**Comment by Arksine** [comment]
Thanks.  Overall it looks ok, the one line needs to be wrapped.  Also, example config should be added [configuration.md](https://github.com/Arksine/moonraker/blob/master/docs/configuration.md#power).

**Comment by Doridian** [comment]
@Arksine Addresses your comments

**Comment by Arksine** [comment]
Thanks.

**Pull request merged**